app/src/main/java/com/github/houseorganizer/houseorganizer/login/LoginEmail.java

```diff
@@ -33,10 +37,8 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private boolean inputsNotEmpty() {
-        EditText email_field = findViewById(R.id.log_enter_email);
-        EditText password_field = findViewById(R.id.log_enter_password);
-        String email = email_field.getText().toString();
-        String password = password_field.getText().toString();
+        String email = ((EditText) findViewById(R.id.log_enter_email)).getText().toString();
+        String password = ((EditText) findViewById(R.id.log_enter_password)).getText().toString();
```

Comment:
not sure you need casts here

Reply:
I think they're necessary because `View` doesn't have a `getText` method

Reply:
i do, otherwise they are typed as View